app/lib/actions/climbing/actions.ts

```diff
@@ -337,20 +338,34 @@ export async function deleteMember(
 
 // Fonction pour supprimer plusieurs membres de la base de données
 export async function deleteMembers(
-  ids: { id: string; imageUrl: string }[],
+  ids: string[],
 ): Promise<{ message: string }> {
-  if (ids.length === 0) {
-    return { message: 'Aucun membre à supprimer.' };
-  }
-
   try {
-    await Promise.all(
-      ids.map(({ id, imageUrl }) => deleteMember(id, imageUrl)),
-    );
+    if (ids.length === 0) {
+      return { message: 'Aucun membre à supprimer.' };
+    }
+    const placeholders = ids.map((_, index) => `$${index + 1}`).join(', ');
+
+    const imageQuery = `SELECT picture FROM members WHERE id IN (${placeholders})`;
+    const imageResults = await sql.query(imageQuery, ids);
+
+    const imageUrls = imageResults.rows
```

Comment:
move this after the await sql.query(deleteQuery, ids);

Reply:
This is not possible. We need the urls before the action of delete members by ids. Otherwise we get an empty array and pictures stay into cloudinary.